packages/office-ui-fabric-react/src/components/GroupedList/examples/GroupedList.CustomCheckbox.Example.tsx

```diff
@@ -37,54 +34,44 @@ export class GroupedListCustomCheckboxExample extends React.Component<{}, IGroup
           fieldName: key,
           minWidth: 300,
         }),
-      );
-    this._groups = createGroups(groupCount, groupDepth, 0, groupCount);
-
-    this._selection = new Selection();
-    this._selection.setItems(this._items);
-
-    this.state = {};
-  }
+      ),
+  );
+  const selection = useConst(() => {
+    const s = new Selection();
+    s.setItems(items);
+    return s;
+  });
```

Comment:
Pretty sure it's possible to just do this now (Selection didn't have an `items` prop in the past, which is why it was creating and updating separately before)
```suggestion
  const selection = useConst(() => new Selection({ items }));
```